### DISCUSSION TITLE
ESPresense to open a gate

### DISCUSSION THREAD

**Discussion by Soopic** [ESPresense/ESPresense#1291] [Q&A] [answered]
Hey everyone, I was wondering if it is secure to use ESPresense to detect my watch when it is about 20 cm away from the ESP, and detect that signal to open a gate in front of my house. I wanted to use NFC but it's not possible with WearOS watches.
Thanks !

**Comment by KurtCobra69** [comment, marked as answer]
Yes, i think it´s posible. I Open my frontdoor when espresence detect my iphone when it is about 0,5m and i push a button.
greetings
